content/en/blog/_posts/2021-12-20-kubernetes-in-kubernetes-and-pxe-bootable-server-farm.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: blog
+title: "Kubernetes-in-Kubernetes and our PXE bootable server farm"
```

Comment:
This did cross my mind
```suggestion
title: "Kubernetes-in-Kubernetes and the WEDOS PXE bootable server farm"
```

I also hope to rustle up someone to review the article @kvaps

Reply:
well, why not